Dockerfile

```diff
@@ -6,17 +6,13 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get -q update && \
     apt-get -q --yes install \
-      mysql-client \
-      nodejs-legacy \
-      npm && \
+      mysql-client && \
```

Comment:
I would recommend a a dev specific one. Then you don't have to worry about mixing Dockerfile logic for different environments, like the `CMD` and `ENTRYPOINT` instructions. You may want to leave these out for a dev container.